### PR TITLE
INTMDB-736: Add python CDK examples

### PR DESCRIPTION
**Pull request by maastha**
## Proposed changes

_Jira ticket:_ [INTMDB-736](https://jira.mongodb.org/browse/INTMDB-736)

Add python CDK examples

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code
- [x] I have tested the CDK constructor in a CFN stack. See [TESTING.md](../TESTING.md)

### CODE REVIEW

File: examples/python/l1-resources/project.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from aws_cdk import (
+  # Duration,
```
Comment (blva) [inline examples/python/l1-resources/project.py:2]: [q] do we need these commented values in the import?

File: examples/python/l2-resources/datadog_integration.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+from awscdk_resources_mongodbatlas import ( DatadogIntegration, DatadogRegion )
+
+class CdkTestAppPython2Stack(Stack):
+
+  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+    super().__init__(scope, construct_id, **kwargs)
+
+    projectt_id = self.node.try_get_context('project_name')
+    apii_key = self.node.try_get_context('org_id')
```
Comment (blva) [inline examples/python/l2-resources/datadog_integration.py:16]: [q] why do we try to get org_id and name it apii_key in l2? (might be lacking context)

File: examples/python/l1-resources/project.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+from awscdk_resources_mongodbatlas import CfnProject
+
+class CdkTestAppPython2Stack(Stack):
+
+  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+    super().__init__(scope, construct_id, **kwargs)
+
+    project_name = self.node.try_get_context('project_name')
+    org_id = self.node.try_get_context('org_id')
+    profile_name = self.node.try_get_context('profile')
```
Comment (colm-quinn) [inline examples/python/l1-resources/project.py:17]: [q] Any error handling here for failures - will CDK give a useful error message or just a python stack trace?

Comment (maastha) [reply examples/python/l1-resources/project.py]: cdk diff fails which is the first step (before cdk deploy), and notifies user of missing required properties

File: examples/python/l1-resources/project.py (hunk 1 above)
Comment (maastha) [reply examples/python/l1-resources/project.py]: removed

File: examples/python/l2-resources/datadog_integration.py (hunk 2 above)
Comment (maastha) [reply examples/python/l2-resources/datadog_integration.py]: typo error when copying from another IDE:( fixed

File: examples/python/l2-resources/datadog_integration.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+from constructs import Construct
+
+from awscdk_resources_mongodbatlas import ( DatadogIntegration, DatadogRegion )
+
+class CdkTestAppPython2Stack(Stack):
+
+  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+    super().__init__(scope, construct_id, **kwargs)
+
+    projectt_id = self.node.try_get_context('project_name')
```
[the remark this answers is not in the document]
Comment (maastha) [reply examples/python/l2-resources/datadog_integration.py]: renamed variables

**Review by blva — APPROVED** [review]
LGTM thanks for the changes

**Review by colm-quinn — APPROVED** [review]
LGTM!

**Pull request merged**